### DISCUSSION TITLE
Version 0.6.0

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#338] [Announcements]
Good day.  Machinaris v0.6.0 is now available with support for many more blockchain forks:

 - [NChain](https://gitee.com/ext9/ext9-blockchain) - cross-farming support for this blockchain fork.
 - [HDDCoin](https://github.com/HDDcoin-Network/hddcoin-blockchain) - cross-farming support for this blockchain fork.
 - [Chives](https://github.com/HiveProject2021/chives-blockchain) - support for this blockchain fork.
 - [Flax](https://github.com/Flax-Network/flax-blockchain) - now farmed in a separtate Docker container.

Core enhancements to underlying components include:
 - [Plotman](https://github.com/ericaltendorf/plotman/pull/937) - enhancement to support plotting for Chives via Madmax.
 - [Chiadog](https://github.com/martomi/chiadog/pull/302) - enhancement to support monitoring of other blockchain forks.
 
 Really big thanks to all those that helped review and provide feedback on this big step forward for Machinaris!

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/0.6.0'>Version 0.6.0</a>.</em>

Upgrades from v0.5.x to 0.6.0:

- [Unraid](https://github.com/guydavis/machinaris/wiki/Unraid#how-do-i-update-from-v05x-to-v060-with-fork-support)
- [Linux](https://github.com/guydavis/machinaris/wiki/Unraid#how-do-i-update-from-v05x-to-v060-with-fork-support)

**Comment by okkar** [comment]
[Edit] Found it: https://github.com/guydavis/machinaris/wiki/Linux#how-do-i-update-from-v05x-to-v060-with-fork-support

Just realised this isn't gonna be a drop-in replacement. :D

What general steps should I take if I'm simply farming Chia + Flax?
